src/components/ArchivedReportFooter.js

```diff
@@ -0,0 +1,91 @@
+import lodashGet from 'lodash/get';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import CONST from '../CONST';
+import Banner from './Banner';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import compose from '../libs/compose';
+import personalDetailsPropType from '../pages/personalDetailsPropType';
+import ONYXKEYS from '../ONYXKEYS';
+
+const propTypes = {
+    /** The reason this report was archived */
+    reportClosedAction: PropTypes.shape({
+        /** Message attached to the report closed action */
+        originalMessage: PropTypes.shape({
+            /** The reason the report was closed */
+            reason: PropTypes.string.isRequired,
+
+            /** (For accountMerged reason only), the email of the previous owner of this report. */
+            oldLogin: PropTypes.string,
+
+            /** (For accountMerged reason only), the email of the account the previous owner was merged into */
+            newLogin: PropTypes.string,
+        }).isRequired,
+    }),
+
+    /** The archived report */
+    report: PropTypes.shape({
+        /** The policy this report is attached to */
+        policyID: PropTypes.string,
+    }).isRequired,
+
+    /** Personal details of all users */
+    personalDetails: PropTypes.objectOf(personalDetailsPropType).isRequired,
+
+    /** The list of policies the user has access to. */
+    policies: PropTypes.objectOf(PropTypes.shape({
+        /** The name of the policy */
+        name: PropTypes.string,
+    })).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    reportClosedAction: {
+        originalMessage: CONST.REPORT.ARCHIVE_REASON.DEFAULT,
+    },
+};
+
+const ArchivedReportFooter = (props) => {
+    const archiveReason = lodashGet(props.reportClosedAction, 'originalMessage.reason', CONST.REPORT.ARCHIVE_REASON.DEFAULT);
+    const policyName = lodashGet(props.policies, `policy_${props.report.policyID}.name`);
+    let displayName = lodashGet(props.personalDetails, `${props.report.ownerEmail}.displayName`, props.report.ownerEmail);
+
+    let oldDisplayName;
+    if (archiveReason === CONST.REPORT.ARCHIVE_REASON.ACCOUNT_MERGED) {
+        const newLogin = props.reportClosedAction.originalMessage.newLogin;
+        const oldLogin = props.reportClosedAction.originalMessage.oldLogin;
+        displayName = lodashGet(props.personalDetails, `${newLogin}.displayName`, newLogin);
+        oldDisplayName = lodashGet(props.personalDetails, `${oldLogin}.displayName`, oldLogin);
+    }
+
+    return (
+        <Banner
+            text={props.translate(`reportArchiveReasons.${archiveReason}`, {
+                displayName,
+                oldDisplayName,
+                policyName,
+            })}
+            shouldRenderHTML={archiveReason !== CONST.REPORT.ARCHIVE_REASON.DEFAULT}
```

Comment:
Do we have other examples where we are storing translated HTML strings? Not that this is a bad idea (I had a similar one recently), but might be a Pandora's box.

Reply:
We don't really have other examples right now, but it seems much cleaner than having `TextPart1`, `TextPart2`, `TextPart3` like we have elsewhere.

Reply:
Right, I guess that's my concern. We're introducing a "new way" to do something. So, maybe we can just keep doing things the "old way" and then improve this later everywhere after getting some consensus on whether we want to have HTML in translations. 

I'm not sure how likely it is, but thought of one way this could get confusing e.g. if someone gets the idea that all translations can have HTML in them by default and then add a `<strong>` element to something that is consumed by something else that _won't_ render HTML and then we end up rendering html text instead of html.

Reply:
That's all valid, but maybe for the sake of urgency we can move forward with the HTML-based solution I have here? I will follow-up in slack to have a conversation around whether HTML in translations is a good practice or not, and we can either refactor this to make it plain-text based, or refactor other translations to use HTML.

Reply:
If we are concerned with urgency it is better to maintain the status quo i.e. do not add HTML to the translations file.

Reply:
Discussing in Slack, but I kinda like this solution: 

>Update this code to pass in `<strong>me@someone.com</strong>` as a parameter that way translations file does not have html added to them

This way it's up to the consumer of the translation stuff to say which parts are HTML or not. And we won't introduce confusion about whether "translations" can have HTML in them or not or whether other consumers would be able to render HTML.

Reply:
Updated! I will try to bring this topic to slack (probably sometime next week) to get broad consensus on a general solution that we can implement everywhere. Maybe `react-native-render-html`, but maybe not.